docs/source/contributing/developing.adoc

```diff
@@ -131,10 +131,9 @@ If you seeing Glide errors during `make vendor`, you can clear the Glide cache v
 == Building Minishift
 
 [[build-minishift-binary]]
-=== Building the Minishift binary
+=== Building the Minishift Binary
```

Comment:
@thatdocslady I mention that they are consistent. as the report would have only pick one of the both cases.